compute/metadata/README.md

```diff
@@ -11,5 +11,6 @@ These samples demonstrate interacting with the Compute Engine metadata service.
 These samples are used on the following documentation page:
 
 > https://cloud.google.com/compute/docs/metadata
+> https://cloud.google.com/compute/docs/instances/verifying-instance-identity
```

Comment:
The two links show up side by side, could you fix the formatting? Reference: https://cloud.google.com/compute/docs/storing-retrieving-metadata